CONTRIBUTING.md

```diff
@@ -155,6 +155,9 @@ let ``preserve compile directive between piped functions, 512`` () = ...
 - Not mandatory, but when fixing a bug consider using `fix-<issue-number>` as the git branch name.<br />
 For example, `git checkout -b fix-1404`.
 
+- Code should be formatted using Fantomas. This command should format all your code:
+  `dotnet fake run build.fsd -t Format`
```

Comment:
build.fs**x**.
Perhaps it is also worth mentioning that `-t FormatChanged` exists.
This formats all the changed files in git.

Reply:
Good idea, I added that